extensions/px-to-tailwind/package.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "px-to-tailwind",
+  "title": "Px to Tailwind",
```

```suggestion
  "title": "Tailwind Size Conversion",
```
Could this be a better naming?